<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackingMenu</class>
 <widget class="QDialog" name="TrackingMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tracking Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>What would you like to track?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Weight">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Calories">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Calories</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Measurements">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Measurements</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Exercises">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Exercises</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Calories">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Exercises">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Measurements">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>280</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
